VPNShield/VPNShield.cs

```diff
@@ -38,12 +38,16 @@ public class VPNShield : Plugin
         public bool autoWhitelistUpdated = false;
         public bool autoBlacklistUpdated = false;
 
-        private readonly string defaultConfig =
+		public string noPurchaseKick;
```

Comment:
This file is inconsistent with tabs/spaces, please convert the file to use space indentation to remove this issue.